.yamato/config/shadergraph_builtin_foundation.metafile

```diff
@@ -0,0 +1,101 @@
+project:
+  name: ShaderGraph_BuiltIn_Foundation
+  folder: BuiltInGraphicsTest_Foundation
+test_platforms:
+  - type: playmode
+  - type: playmode
+    name: playmode_cache
+    utr_flags:
+    - [all]: --extra-editor-arg="-adb2"
+    - [all]: --extra-editor-arg="-enableCacheServer"
+    - [all]: --extra-editor-arg="-cacheServerEndpoint" --extra-editor-arg="cacheserver-slo.hq.unity3d.com"
+    - [all]: --extra-editor-arg="-cacheServerNamespacePrefix" --extra-editor-arg="SRP"
+    - [all]: --extra-editor-arg="-cacheServerEnableDownload" --extra-editor-arg="true"
+    - [all]: --extra-editor-arg="-cacheServerEnableUpload" --extra-editor-arg="true"
+expression_trigger:
+  expression: 'pull_request.target eq "<TARGET_BRANCH>" AND NOT pull_request.draft AND pull_request.changes.any match ["com.unity.shadergraph/**/*", "TestProjects/BuiltInGraphicsTest_Foundation/**/*"] AND NOT pull_request.push.changes.all match ["*.md", "**/*.md", "doc/**/*", "**/Documentation*/**/*"]'
+platforms:
+  - name: Win
+    apis:
+      - name: DX11
+      - name: Vulkan
+    build_configs:
+      - name: mono
+    color_spaces:
+      - Linear
+    agents_project:
+      default:
+        type: Unity::VM::GPU
+        image: sdet/gamecode_win10:stable
+        flavor: b1.xlarge
+        model: rtx2080
+  - name: OSX
+    apis:
+      - name: Metal
+      - name: OpenGLCore
+    build_configs:
+      - name: mono
+    color_spaces:
+      - Linear
+  - name: Linux
+    apis:
+      - name: OpenGLCore
+      - name: Vulkan
+    build_configs:
+      - name: mono
+    color_spaces:
+      - Linear
+pr:
+  dependencies:
+    - platform: Win
+      api: DX11
+      build_config: mono
+      color_space: Linear
+      test_platforms:
+        - playmode_cache
+    - platform: OSX
+      api: OpenGLCore
+      build_config: mono
+      color_space: Linear
+      test_platforms:
+        - playmode_cache
+    - platform: Linux
+      api: Vulkan
+      build_config: mono
+      color_space: Linear
+      test_platforms:
+        - playmode_cache
+    - platform: OSX
+      api: Metal
+      build_config: mono
+      color_space: Linear
+      test_platforms:
+        - playmode_cache
+nightly:
+  dependencies:
+    - project: ShaderGraph_BuiltIn_Foundation
+      pr: true
+    - platform: Win
+      api: DX11
+      build_config: mono
+      color_space: Linear
+      test_platforms:
+        - playmode_cache
+    - platform: OSX
+      api: OpenGLCore
+      build_config: mono
+      color_space: Linear
+      test_platforms:
+        - playmode_cache
+    - platform: Linux
+      api: Vulkan
+      build_config: mono
+      color_space: Linear
+      test_platforms:
+        - playmode_cache
+    - platform: OSX
+      api: Metal
+      build_config: mono
+      color_space: Linear
+      test_platforms:
+        - playmode_cache
```

Comment:
You can safely remove these lines as the playmode_cache pipelines are already included in the PR pipeline.
If not, it duplicates https://github.com/Unity-Technologies/Graphics/pull/4030/files#diff-d36729fd8bb4b83fa670708737b0c9c5629e398f67808429b76a47baffdd95e9R29

Reply:
So just leave lines 74-77?

Reply:
yes, that remove the duplicates in the nightly job :)

Reply:
Latest push should address this.